go-controller/pkg/syncmap/syncmap.go

```diff
@@ -140,6 +140,15 @@ func (c *SyncMap[T]) LoadOrStore(lockedKey string, newEntry T) (value T, loaded
 	}
 }
 
+// Store sets the value for a key.
+// loaded return value signals if the object was present.
```

Comment:
that probably doesn't apply to this function? :)

Reply:
cut&paste. my fault! :disappointed: